ehrql/__main__.py

```diff
@@ -208,18 +208,18 @@ def add_create_dummy_tables(subparsers, environ, user_args):
         "create-dummy-tables",
         help=strip_indent(
             """
-            Generate dummy data for a dataset and write out tables as CSV.
+            Generate dummy tables and write them out as CSV files (one per table).
```

Comment:
nit: do you write _to_ CSV files or _as_ CSV files?

Reply:
I don't know, but previously we used "as". Do you have a preference?

Reply:
Not changed.

Reply:
Fine :)